### ISSUE TITLE
Specify a set of regex patterns for text that should be translated dynamically

### ISSUE THREAD

**Issue by lloyddewit** [IDEMSInternational/TranslateWinForms#4]
This issue addresses a comment in PR #3.

Function `GetCsvText()` uses various regex patterns to determine if a piece of text should be translated statically or dynamically.

These patterns work correctly for R-Instat but probably won't apply to other products that may potentially use this library. It's not a problem for Climsoft because Climsoft builds its database differently and doesn't use this function.

However, these patterns belong with the product, not the library.

We should change this function so that it accepts a list of regex patterns to chack against.

We also need to change R-Instat so that it passes this list to the TranslateWinForms library.

In summary, we should:

- Keep the check for multiline text,
- Keep the check for text that doesn't contain any letters
- Move the other checks to the product level

**Comment by Patowhiz** [comment]
In climsoft we also have some list views and grids that have column headers and also dynamic contents that need to be translated. The regex patterns or heuristics needs to factor such scenario.
